src/AvaloniaEdit/TextEditorOptions.cs

```diff
@@ -719,5 +719,30 @@ public CompletionAcceptAction CompletionAcceptAction
                 }
             }
         }
+
+        // The default LineHeightFactor matches the line height in the Visual Studio text editor.
+        private const double DefaultLineHeightFactor = 1.0455;
```

Comment:
I tested the changes and I still see the text quite compressed using the default `1.0455 `value. I suggest using a value that ensures the text is displayed in the same height before #537.
```suggestion
        private const double DefaultLineHeightFactor = 1.16;
```

See attached image 
<img width="1112" height="1049" alt="image" src="https://github.com/user-attachments/assets/b8da9e9c-76a4-43e7-8280-9bca723f8357" />